wiki/Manual.md

```diff
@@ -4606,16 +4606,17 @@ $params = {
 $order = $exchange->create_order ('ETH/USDT', 'market', 'buy', 0.1, 1500, $params)
 ```
 <!-- tabs:end -->
-Typically, it means to touch the price from **any direction**. However, if some exchanges require that you provided `triggerDirection`, with either `above` or `below` values:
+###### Trigger Direction
+Typically, it means to touch the price from **any direction**. However, if some exchanges require that you provided `triggerDirection`, with either `ascending` or `descending` values:
```

Comment:
this being uner Triggerdirection is confusing, because that's not the case when we have a triggerdirection

Reply:
is this last edit ok?